src/status_im/ethereum/transactions/core.cljs

```diff
@@ -273,7 +260,7 @@
 
                           (seq transfers)
                           (concat
```

Comment:
concat is not needed anymore

Reply:
emm.. if replace `(concat [] (mapv add-transfer transfers))` to `(mapv add-transfer transfers)` will get error:
```
 ERROR  Error: function status_im$ethereum$transactions$core$add_transfer(var_args){
var G__86827 = arguments.length;
switch (G__86827) {
case 1:
return status_im.ethereum.transactions.core.add_transfer.cljs$core$IFn$_invoke$arity$1((arguments[(0)]));

break;
case 2:
return status_im.ethereum.transactions.core.add_transfer.cljs$core$IFn$_invoke$arity$2((arguments[(0)]),(arguments[(1)]));

break;
default:
throw (new Error(["Invalid arity: ",cljs.core.str.cljs$core$IFn$_invoke$arity$1(arguments.length)].join('')));

}
} is not ISeqable
``` @flexsurfer

Reply:
that's strange

Reply:
🤷‍♂️